links.html

```diff
@@ -40,13 +48,16 @@ <h1>Useful Links</h1>
 			<a href="https://caniuse.com/">Cansius.com</a>
 			</li>
 			<li>
-			<a href="https://necolas.github.io/normalize.css/8.0.1/normalize.css">normalize</a>
+			<a href="https://necolas.github.io/normalize.css/8.0.1/normalize.css" target="_blank">normalize</a>
+			</li>
+			<li>
+			<a href="https://fontpair.co" target="_blank">fontPair</a>
 			</li>
 			<li>
-			<a href="https://fontpair.co">fontPair</a>
+			<a href="https://fonts.google.com/" target="_blank">google fonts</a>
 			</li>
 			<li>
-			<a href="https://fonts.google.com/">google fonts</a>
+			<a href="https://realfavicongenerator.net/" target="_blank">Favicon Generator</a>
 			</li>
```

Comment:
A couple missing links, and don't forget the target attribute when you need to open links in a new tab